src/checkers/inference/solver/util/ConstantUtils.java

```diff
@@ -0,0 +1,57 @@
+package checkers.inference.solver.util;
+
+import org.checkerframework.javacutil.AnnotationUtils;
+
+import java.util.Collection;
+
+import javax.lang.model.element.AnnotationMirror;
+
+import checkers.inference.model.ComparableConstraint;
+import checkers.inference.model.ConstantSlot;
+import checkers.inference.model.Constraint;
+import checkers.inference.model.EqualityConstraint;
+import checkers.inference.model.InequalityConstraint;
+import checkers.inference.model.SubtypeConstraint;
+import checkers.inference.solver.frontend.Lattice;
+
+/**
+ * Utils methods for constant slot.
+ * 
+ * @author jianchu
+ *
+ */
+public class ConstantUtils {
```

Comment:
Maybe change the name to `ConstantSlotUtils`? At first time I saw this class, I though it was a util class for operating `Constant` enum class...